swagger.json

```diff
@@ -4508,6 +4624,27 @@
             }
           }
         },
+        "HosCertifiedShiftResponse": {
+            "type": "object",
+            "properties": {
+                "dayStartMs": {
+                    "type": "integer",
+                    "format": "int64",
+                    "description": "Unix epoch time (in ms) of time of when this certified shift starts",
+                    "example": 1462881998034
+                },
+                "shippingDocIds": {
+                    "type": "string",
+                    "description": "Customer shipping document IDs, potentially multiple numbers delimited by a comma or space",
+                    "example": "1059984"
+                },
+                "trailerNames": {
+                    "type": "string",
+                    "description": "Trailer number, potentially multiple names delimited by a comma or space",
```

Comment:
These descriptions differ from the descriptions in `hosCertifiedShiftParams`, we should potentially standardize. If not, I think using "Trailer number" in this description is confusing, since the name of the field is "trailerNames".

Reply:
You're right here, I keep shifting back and forth between names and I don't catch some of them

Reply:
This description also mentions both `number` and `multiple names`.